x/concentrated-liquidity/keeper.go

```diff
@@ -99,3 +99,12 @@ func (k *Keeper) SetListeners(listeners types.ConcentratedLiquidityListeners) *K
 
 	return k
 }
+
+// ValidatePermissionlessPoolCreationEnabled returns nil if permissionless pool creation in the module is enabled.
+// Otherwise, returns an error.
+func (k Keeper) ValidatePermissionlessPoolCreationEnabled(ctx sdk.Context) error {
```

Comment:
nit:consider changing to
```suggestion
func (k Keeper) IsPermissionlessPoolCreationEnabled(ctx sdk.Context) error {
```
For code style unification

Reply:
ref: https://github.com/osmosis-labs/osmosis/pull/4952#discussion_r1170726335